Python/ceval_macros.h

```diff
@@ -151,17 +151,15 @@ GETITEM(PyObject *v, Py_ssize_t i) {
     } while (0)
 #define JUMPTO(x)       do { \
```

Comment:
This also has only one call site left (in exception_unwind), so maybe expand there?